middleware/index.ts

```diff
@@ -1,4 +1,5 @@
-
+import * as fs from 'fs';
+import {LoggerLevel} from '@salesforce/core'
```

Comment:
nit: inconsistent trailing `;` chars - I think most of this source *does* use semicolons?

Reply:
I should add some linting to this project...